app/views/browse/mobile-clients.html

```diff
@@ -0,0 +1,57 @@
+<div class="middle">
+  <div class="middle-header">
+    <div class="container-fluid">
+      <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+      <alerts alerts="alerts"></alerts>
+      <div>
+        <h1 class="contains-actions">
+          <div class="pull-right dropdown" ng-if="mobileClient">
+            <button type="button" class="dropdown-toggle btn btn-default actions-dropdown-btn hidden-xs" data-toggle="dropdown">
+              Actions
+              <span class="caret" aria-hidden="true"></span>
+            </button>
+            <a href=""
+               class="dropdown-toggle actions-dropdown-kebab visible-xs-inline"
+               data-toggle="dropdown"><i class="fa fa-ellipsis-v"></i><span class="sr-only">Actions</span></a>
+            <ul class="dropdown-menu dropdown-menu-right actions action-button">
+              <!--TODO add canI filter, needs an update to AuthorisationService-->
+              <li>
+                <delete-link
+                  kind="MobileClient"
+                  group="mobile.k8s.io"
+                  resource-name="{{mobileClient.metadata.name}}"
+                  project-name="{{mobileClient.metadata.namespace}}"
+                  alerts="alerts"
+                  redirect-url={{redirectUrl}}>
+                </delete-link>
+              </li>
+            </ul>
+          </div>
+          {{mobileClient.spec.name}}
+          <small class="meta" ng-if="mobileClient">created <span am-time-ago="mobileClient.metadata.creationTimestamp"></span></small>
+        </h1>
+        <labels labels="mobileClient.metadata.labels" clickable="true" kind="mobileClient" project-name="{{mobileClient.metadata.namespace}}" limit="3"></labels>
+      </div>
+    </div>
+  </div>
+  <div class="middle-content" persist-tab-state>
+    <div class="container-fluid">
+      <div ng-if="!loaded">Loading...</div>
+      <div class="row">
+        <div class="col-md-12" ng-class="{ 'hide-tabs' : !mobileClient }">
+          <uib-tabset>
+            <uib-tab active="selectedTab.configuration">
+              <uib-tab-heading>Configuration</uib-tab-heading>
+              <pre>
+                {{mobileClient | json}}
```

Comment:
what formatting, indentation is used for this json filter?

Reply:
this is built in filter into angular